src/identity/PublicIdentity.ts

```diff
@@ -15,6 +15,17 @@ export interface IURLResolver {
 }
 
 export default class PublicIdentity implements IPublicIdentity {
+  /**
+   * @description (STATIC) Creates a new Public Identity from a DID (Decentralised identifier) object.
+   * @param didDocument Public keys and pseudonymous biometrics
```

Comment:
Great that you added a definition, yet to me it doesn't describe what a didDocument is (also I'm not sure what biometrics have to do with DIDs, at least in the base case?).  
To see what a didDocument is, the IPublicIdentity test file and the official did spec might be good places to look.

Reply:
changed it up, what do you think now?